tests/integration_tests/test_load.py

```diff
@@ -142,10 +127,16 @@ async def test_backpressure_handling(lobby_server, caplog):
     proto.writer.transport.set_write_buffer_limits(high=0)
     proto.reader._limit = 0
 
+    # TRACE will be spammed with thousands of messages
+    caplog.set_level(logging.DEBUG)
+
+    # It takes quite a lot of spamming for the backpressure handling to take
+    # affect.
```

Comment:
Can you point me to the code that this is supposed to trigger?

Reply:
Basically, it should cause the line:
```python3
await connection.on_message_received(message)
```
in `ServerContext.client_connected` to block indefinitely. There isn't necessarily any single handler, but if 'protocol.drain' is not called between each request-response then this test will fail.

If you want to read the blog post (essay?) that introduced me to backpressure here it is. Warning though, it's quite long.
https://vorpus.org/blog/some-thoughts-on-asynchronous-api-design-in-a-post-asyncawait-world/

(Also just noticed it should be 'effect' not 'affect' :P. English is hard)